src/components/vote/DelegatesListItem.tsx

```diff
@@ -20,6 +19,8 @@ export const DelegatesListItem = ({
     onSelected: (delegate?: string) => void;
 }) => {
     const { t } = useTranslation();
+    const username = delegate.username() || delegate.address();
```

Comment:
@patricio0312rev 🤔I was about to comment that if the username came from the address, we should truncate it in the middle, but since the extension does not use the new Mainsail network yet, we should always have a username.

I think that, for now, it is better to remove the address part. That will force us to think about it once we implement the Mainsail changes which I suppose we are going to include eventually.

Better to double-check with @ItsANameToo

```suggestion
    const username = delegate.username()
```